src/contract.rs

```diff
@@ -206,6 +216,7 @@ fn add_validator(
     deps: DepsMut,
     info: MessageInfo,
     validator_addr: Addr,
+    account_addr: Addr, // account address of the validator
```

Comment:
Make this a string.

Verify using deps.api.addr_validate() on this string.

Reply:
Let's also call this a wallet_address

Reply:
Renamed account address to wallet_address and converted it from Addr to String and used deps.api.validate to validate the string wherever it is required.